catalog/spatial/registry/registry-rest-endpoint/src/main/resources/templates/summary.hbt

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta content="text/html; charset=windows-1252" http-equiv="content-type"/>
+    <!-- styles -->
+    <link href="/services/registries/assets/css/bootstrap.min.css" rel="stylesheet"/>
+    <style type="text/css">
```

Comment:
Yes, moved everything into a report.css class and linked it in each handlebar file